README.md

```diff
@@ -60,7 +60,9 @@ init` followed by `git submodule update`.
 
 ## Using c4core in your project
 
-c4core is built with cmake, and assumes you also use cmake. Although c4core
+### CMake
+
+c4core is officially built with cmake. Although c4core
```

Comment:
Suggestion: don't make this section about cmake, and reword the first phrase to "c4core can be built with cmake or bazel, or be used header only with the [amalgamation tool](tools/amalgamate.py)". Keep the bazel section as-is.